Sources/PackageGraph/ModulesGraph+Loading.swift

```diff
@@ -336,6 +336,37 @@ fileprivate extension ResolvedProduct {
     }
 }
 
+private func getPathInGraph(
```

Comment:
nit: this would benefit from a doc comment that clarifies the purposes and inputs/outputs of the function.